core/models/Operator.js

```diff
@@ -5,11 +5,13 @@ module.exports = {
     properties: {
       id: {
         type: "integer",
+        unique: true,
```

Comment:
That is not necessary since you set the id as a primary key in table construction. Don't worry, I'll remove this line, but please don't add this into id properties in future.